packages/bot-web-ui/src/components/dashboard/joyride-config.tsx

```diff
@@ -506,49 +505,58 @@ export const DBOT_ONBOARDING_MOBILE = [
         step_key: 0,
     },
     {
-        header: localize('Build from scratch'),
-        img: getImageLocation('dbot-mobile-onboarding-step-1.gif'),
-        content: [
-            localize(
-                'Import a bot from your mobile device or from Google drive, see a preview in the bot builder, and start trading by running the bot, or choose from our pre-made Quick Strategies. '
-            ),
-        ],
+        header: localize('Import or choose your bot'),
+        media: getUrlBase('/public/videos/dbot-mobile-onboarding-step-1.mp4'),
+        content: [localize('Import your bot or tap Quick Strategies to choose from the ready-to-use bot templates.')],
         key: 2,
         step_key: 1,
     },
     {
         header: localize('Monitor the market'),
-        img: getImageLocation('dbot-mobile-onboarding-step-2.png'),
+        media: getUrlBase('/public/videos/dbot-mobile-onboarding-step-2.mp4'),
```

Comment:
could be for such cases `const url_path_videos: string = '/public/videos/dbot-mobile-onboarding-step-';`
but it's up to you.
```suggestion
        media: getUrlBase(`${url_path_videos}2.mp4`),
```

Reply:
@maryia-matskevich-deriv i think that wouldn't be necessary as the path is just `/public/videos` and then the file name. If it was a long path then would make this change, but in this case i would like to keep it.